### PR TITLE
Add accountsservice files for Greeter use

### PR DESCRIPTION
**Pull request by codygarver**

### CODE REVIEW

**Comment by codygarver** [comment]
Ready for review but BLOCKED until it's confirmed this is what needs to be removed from Greeter, so the packages won't collide after build.

**Comment by decathorpe** [comment]
Why is this stuff getting moved from greeter to default settings?

**Comment by danrabbit** [comment]
@decathorpe To avoid the cyclicle dependency situation we previously had with wingpanel-indicator-datetime that you had brought up ;)

**Comment by decathorpe** [comment]
> @decathorpe To avoid the cyclical dependency situation we previously had with wingpanel-indicator-datetime that you had brought up ;)

:thinking: Wait ... but the greeter is the application actually providing these services. And now you're moving the files from greeter to default settings, with default settings mandating the use of greeter and the greeter depending on the default settings? The way I see it, you're *introducing* a circular dependency where there was none ...

**Comment by danrabbit** [comment]
@decathorpe No default settings isn't relying on the greeter. It's just providing an accountsservice extension. I think this extension probably needs to be namespaced in a different way to not reference the greeter since it'll be used by other OS components as well.

This way the date time indicator, the system settings pane, the greeter, and granite can all rely on default settings instead of pointing at each other

File: accountsservice/io.elementary.pantheon.AccountsService.policy [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<policyconfig>
+  <action id="io.elementary.pantheon.AccountsService.ModifyOwn">
+    <description gettext-domain="io.elementary.pantheon">Set properties of own user</description>
```
Comment (tintou) [inline accountsservice/io.elementary.pantheon.AccountsService.policy]:
```suggestion
    <description>Set properties of own user</description>
```
We don't have translations and don't need any here

File: accountsservice/io.elementary.pantheon.AccountsService.xml [hunk 2]
```diff
@@ -1,16 +1,16 @@
 <node>
-  <interface name="io.elementary.greeter.AccountsService">
+  <interface name="io.elementary.pantheon.AccountsService">
```
Comment (cassidyjames) [inline accountsservice/io.elementary.pantheon.AccountsService.xml]: @danrabbit would `desktop` make more sense here? That seems to be the convention elsewhere for Pantheon stuff (`io.elementary.desktop`).

Comment (tintou) [reply accountsservice/io.elementary.pantheon.AccountsService.xml]: From slack: Well, Pantheon is the name of the desktop, we can't change that at this point so either make sense to me but I'm more in favor of io.elementary.pantheon as it is a session-wide setting and not a "desktop-component" setting (which is a subtle nuance)

**Comment by tintou** [comment]
It doesn't collide with the greeter so it's safe to merge

**Pull request merged**